app/services/foreman_rh_cloud/upload_hits.rb

```diff
@@ -0,0 +1,51 @@
+module ForemanRhCloud
+  class UploadHits
```

Comment:
Since we do `.new` what do you think about naming it HitsUploader?
```suggestion
  class HitsUploader
```